### ISSUE TITLE
Add Churn dataset to dataset module

### ISSUE THREAD

**Issue by lwgray** [DistrictDataLabs/yellowbrick-datasets#4]
We are in the process of implementing plot directives for all documentation; essentially eliminating the scripts that generated images for the visualizer docs.  The implementation involves utilizing the new Dataset module.  However, the DiscriminationThreshold docs uses the Churn dataset which isn't included in the dataset module.  So the task of this issue is to add in the Churn dataset.

https://github.com/DistrictDataLabs/yellowbrick/blob/80169ab4ac8b066abd4c5f4f8f3af34fc2c76d20/docs/api/classifier/threshold.py#L13-L17

**Comment by bbengfort** [comment]
@lwgray so I have some bad news - I spent the evening looking around for this dataset, and while I found it on Kaggle and something called BigML.com - I couldn't find a publically available dataset with a citation that we could use. I also found a similar dataset on IBM's Watson page, but again - I'm not sure that's freely available to use in the way we are. 

Because the churn image is not used in the documentation (I think it's just the spam dataset right?) Can we get away with omitting this and removing the figure from our docs?
